server/enterprise/message_export/worker.go

```diff
@@ -385,6 +398,13 @@ func extractJobData(logger *mlog.Logger, strmap map[string]string) (data shared.
 	// previous batches).
 	data.ExportPeriodStartTime = data.BatchStartTime
 
+	// JobStartTime is the start of this job. It is different from ExportPeriodStartTime because JobStartTime won't change
+	// if the job processes some batches, is stopped, and picked up again.
+	data.JobStartTime, err = strconv.ParseInt(strmap[JobDataStartTimestamp], 10, 64)
+	if err != nil {
+		return
+	}
```

Comment:
Unrelated but I am thinking that if any of these were to fail it would be quite hard to understand what exactly went wrong since we are not wrapping the error.